ovs/lib/albacontroller.py

```diff
@@ -63,7 +63,11 @@ class AlbaController(object):
     ETCD_NR_OF_AGENTS_KEY = ETCD_MAINTENANCE_KEY + '/nr_of_agents'
 
     @staticmethod
-    def _get_abm_service_name(alba_backend):
+    def get_abm_service_name(alba_backend):
+        """
```

Comment:
Why do we even use this function?
Its stupid imo, since the exact name is in the abm services linked to the backend
So there is no need for a + '-abm'